app/views/user_mailer/password_reset.text.erb

```diff
@@ -1,4 +1,4 @@
-
-User#password_reset
-
-<%= @greeting %>, find me in app/views/user_mailer/password_reset.text.erb
+<%= t ".content1" %>
```

Comment:
toàn bộ chỗ này gộp thành 1 cái i18n nhé, các chỗ khác mà làm đc tương tự thì cũng làm theo luôn nha